db/raw/master-head.sql

```diff
@@ -475,6 +447,506 @@ CREATE TABLE tracking (
 
 ALTER TABLE public.tracking OWNER TO m13;
 
+SET search_path = sqitch, pg_catalog;
```

Comment:
Hm, I think I get it now. The first time you ran those commands it was likely before you had initiated sqitch - now the m13 database has a `changes` table which gets included in the raw files.

...we'll have to rework some stuff, I think.